server/src/main/java/org/apache/iotdb/db/mpp/plan/expression/multi/FunctionExpression.java

```diff
@@ -512,6 +512,7 @@ private UDFQueryTransformer constructUdfTransformer(
 
   @Override
   public boolean isMappable(TypeProvider typeProvider) {
+    if (isBuiltInAggregationFunctionExpression) return true;
```

Comment:
It is better to add a pair of parentheses, even if there is only one statement inside the if

Reply:
```suggestion
    // For calculating nested expressions of BuiltinAggregationFunction
    if (isBuiltInAggregationFunctionExpression) {return true};
```